lldb/include/lldb/Target/StopInfo.h

```diff
@@ -79,6 +79,11 @@ class StopInfo : public std::enable_shared_from_this<StopInfo> {
 
   virtual bool IsValidForOperatingSystemThread(Thread &thread) { return true; }
 
+  /// A Continue operation can result in a false stop event
+  /// before any execution has happened in certain cases on Darwin.
+  /// We need to silently continue again time.
```

Comment:
"again time" -> "again one more time"?

Also, this isn't a Darwin specific feature, on any system where the Continue might stall, you could use this.  I'd describe what it does rather than primarily refer to Darwin.